content/c-sharp/concepts/switch/switch.md

```diff
@@ -0,0 +1,79 @@
+---
+Title: 'Switch'
+Description: 'Switch statements execute a code block from a list of conditions if a condition is true.'
+Subjects:
+  - 'Code Foundations'
+  - 'Web Development'
+  - 'Mobile Development'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Control Flow'
+  - 'Switch'
+  - 'C#'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+**Switch** statements execute a code block from a list of `case` conditions for which the `expression` validates as `true`. Switch statements are similar to if/else statements.
+
+## Syntax
+
+```pseudo
+switch (expression) 
+{
+  case alpha:
+    // Code block
+    break;
+  case beta:
+    // Code block
+    break;
+  default:
+    // Code block
+    break;
+}
+```
+
+A `switch` statement includes one or more `case` conditions. The `case` conditions are evaluated in top-to-bottom order. If none of the `case` conditions evaluate to `true`, the `default` condition is executed. If no `default` is defined, the switch statement is bypassed.
+
+## Properties
```

Comment:
Remove this header, these aren't actually "properties" and they're elaborating on the syntax. I'd replace this with a sentence introducing the bulleted list.

Reply:
Updated